### ISSUE TITLE
Movement handles do not appear

### ISSUE THREAD

**Issue by metalfoxdev** [mltframework/shotcut#1405]
The movement handles for filters such as "Size, Position and Rotate", "Text: Simple" or "Text: Rich" do not appear on screen.

Details:

Shotcut 22.12.21 (Flatpak from Flathub)
Manjaro Linux (KDE Plasma 5.26.5, Qt Version 5.15.8, Kernel Version 6.1)

**Comment by ddennedy** [comment]
Others in the forum reported it too, but I do not reproduce it in Flatpak on Pop!_OS 22.04 with AMD Radeon hardware. This part is using high level Qt QML Quick, which generates OpenGL. So, it is likely some OpenGL compatibility issue over which we have no direct control. There is a migration to next major version 6 of Qt. That will likely make a difference.

**Comment by metalfoxdev** [comment]
Will the QT 6 version be rolled out to flathub?

**Comment by ddennedy** [comment]
Yes, but not the main flatpak until the next release planned by end of April. However, our development branch is already looking good, and you can try it out!
https://github.com/flathub/org.shotcut.Shotcut/pull/69

**Comment by metalfoxdev** [comment]
Thank you! I will try the test build and see whether or not the issue is fixed. If the issue is fixed then I will close this issue. If the issue still persists then I will keep this open and share my findings while trying to fix it. Thank you for your help!

**Comment by ddennedy** [comment]
Related forum posts:

https://forum.shotcut.org/t/text-input-rectangle-invisible-no-way-to-enter-text/37510

https://forum.shotcut.org/t/linux-intel-some-ui-elements-not-showing/37839
